src/arch/helperadvsimd.h

```diff
@@ -90,137 +90,137 @@ static INLINE vfloat vgather_vf_p_vi2(const float *ptr, vint2 vi2) {
-static INLINE vfloat vmlanp_vf_vf_vf_vf(vfloat x, vfloat y, vfloat z) { return vsub_vf_vf_vf(z, vmul_vf_vf_vf(x, y)); }
+static INLINE VECTOR_CC vfloat vmla_vf_vf_vf_vf(vfloat x, vfloat y, vfloat z) { return vadd_vf_vf_vf(vmul_vf_vf_vf(x, y), z); }
+static INLINE VECTOR_CC vfloat vmlanp_vf_vf_vf_vf(vfloat x, vfloat y, vfloat z) { return vsub_vf_vf_vf(z, vmul_vf_vf_vf(x, y)); }
 #endif
 
 // |x|, -x
-static INLINE vfloat vabs_vf_vf(vfloat f) { return vabsq_f32(f); }
-static INLINE vfloat vneg_vf_vf(vfloat f) { return vnegq_f32(f); }
+static INLINE VECTOR_CC vfloat vabs_vf_vf(vfloat f) { return vabsq_f32(f); }
+static INLINE VECTOR_CC vfloat vneg_vf_vf(vfloat f) { return vnegq_f32(f); }
 
 // max, min
-static INLINE vfloat vmax_vf_vf_vf(vfloat x, vfloat y) {
+static INLINE VECTOR_CC vfloat vmax_vf_vf_vf(vfloat x, vfloat y) {
   return vmaxq_f32(x, y);
 }
-static INLINE vfloat vmin_vf_vf_vf(vfloat x, vfloat y) {
+static INLINE VECTOR_CC vfloat vmin_vf_vf_vf(vfloat x, vfloat y) {
   return vminq_f32(x, y);
 }
 
 // Comparisons
-static INLINE vmask veq_vm_vf_vf(vfloat x, vfloat y) { return vceqq_f32(x, y); }
-static INLINE vmask vneq_vm_vf_vf(vfloat x, vfloat y) {
+static INLINE VECTOR_CC vmask veq_vm_vf_vf(vfloat x, vfloat y) { return vceqq_f32(x, y); }
+static INLINE VECTOR_CC vmask vneq_vm_vf_vf(vfloat x, vfloat y) {
   return vmvnq_u32(vceqq_f32(x, y));
 }
-static INLINE vmask vlt_vm_vf_vf(vfloat x, vfloat y) { return vcltq_f32(x, y); }
-static INLINE vmask vle_vm_vf_vf(vfloat x, vfloat y) { return vcleq_f32(x, y); }
-static INLINE vmask vgt_vm_vf_vf(vfloat x, vfloat y) { return vcgtq_f32(x, y); }
-static INLINE vmask vge_vm_vf_vf(vfloat x, vfloat y) { return vcgeq_f32(x, y); }
+static INLINE VECTOR_CC vmask vlt_vm_vf_vf(vfloat x, vfloat y) { return vcltq_f32(x, y); }
+static INLINE VECTOR_CC vmask vle_vm_vf_vf(vfloat x, vfloat y) { return vcleq_f32(x, y); }
+static INLINE VECTOR_CC vmask vgt_vm_vf_vf(vfloat x, vfloat y) { return vcgtq_f32(x, y); }
+static INLINE VECTOR_CC vmask vge_vm_vf_vf(vfloat x, vfloat y) { return vcgeq_f32(x, y); }
 
 // Conditional select
-static INLINE vfloat vsel_vf_vm_vf_vf(vmask mask, vfloat x, vfloat y) {
+static INLINE VECTOR_CC vfloat vsel_vf_vm_vf_vf(vmask mask, vfloat x, vfloat y) {
   return vbslq_f32(mask, x, y);
 }
 
 // int <--> float conversions
-static INLINE vint2 vtruncate_vi2_vf(vfloat vf) { return vcvtq_s32_f32(vf); }
-static INLINE vfloat vcast_vf_vi2(vint2 vi) { return vcvtq_f32_s32(vi); }
-static INLINE vint2 vcast_vi2_i(int i) { return vdupq_n_s32(i); }
-static INLINE vint2 vrint_vi2_vf(vfloat d) {
-  return vcvtq_s32_f32(vrndnq_f32(d));
+static INLINE VECTOR_CC vint2 vtruncate_vi2_vf(vfloat vf) { return vcvtq_s32_f32(vf); }
+static INLINE VECTOR_CC vfloat vcast_vf_vi2(vint2 vi) { return vcvtq_f32_s32(vi); }
+static INLINE VECTOR_CC vint2 vcast_vi2_i(int i) { return vdupq_n_s32(i); }
+static INLINE VECTOR_CC vint2 vrint_vi2_vf(vfloat d) {
+  return vcvtq_s32_f32(vrndaq_f32(d));
```

Comment:
```
```suggestion
  return vcvtq_s32_f32(vrndaq_f32(d));
This line should be
```
return vcvtq_s32_f32(vrndnq_f32(d));
```